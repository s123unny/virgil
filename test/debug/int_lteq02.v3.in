b ../core/int_lteq02.v3 4
b ../core/int_lteq02.v3 5
run
bt
info l
c
q
