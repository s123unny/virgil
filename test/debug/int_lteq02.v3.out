breakpoint|../core/int_lteq02.v3 4|0
breakpoint|../core/int_lteq02.v3 5|1
stop|stopOnBreakpoint
bt|int_lteq02.main|../core/int_lteq02.v3|4
variable|0|this|int_lteq02|int_lteq02|false
variable|1|a|0|int|false
variable|2|b|0|int|false
stop|end
result|true
